### DISCUSSION TITLE
关于头文件路径的问题

### DISCUSSION THREAD

**Discussion by zhengsq23** [arcsysu/SYsU-lang#6] [Q&A]
lexer将预处理后的代码输入，但是类似# 1 "<stdin>" # 1 "<built-in>" 1
中的内容却无法使用正则表达式匹配得到当前的头文件，应该如何从输入获取当前的头文件路径

**Comment by wu-kan** [comment]
抱歉我没有理解你的问题， 和[这个问题](https://github.com/arcsysu/SYsU-lang/discussions/2)的区别在哪里？
